### PR TITLE
feat(1765): remote events

### PR DESCRIPTION
**Pull request by jhackett1**
fixes: #1765

### CODE REVIEW

File: app/controllers/admin/events_controller.rb [hunk 1]
```diff
@@ -74,7 +74,7 @@ def set_event
 
   def event_params
     params.require(:event).permit(
-      :name, :slug, :date_and_time, :begins_at, :ends_at, :description, :info, :schedule, :venue_id, :external_url,
+      :remote, :name, :slug, :date_and_time, :begins_at, :ends_at, :description, :info, :schedule, :venue_id, :external_url,
```
Comment (matyikriszta) [inline app/controllers/admin/events_controller.rb:77]: @jhackett1 can we rename this field to `virtual` please? So it aligns with the name of the field on `Workshop`.

Comment (jhackett1) [reply app/controllers/admin/events_controller.rb]: done

**Comment by matyikriszta** [comment]
> the remaining codeclimate issues _seem_ low priority to me. i can invest some more time in fixing them, but tbh i don't _think_ i caused them—they don't seem to be attached to any code i wrote or changed, even though they're showing up as "new" in the report.
> 
> lemme know what you think @KimberleyCook

@jhackett1 you are right, I have marked those issues for now as won't fix, they are not relevant to this change.

**Review by matyikriszta — COMMENTED** [review]
@jhackett1 can you add an integration test for this change?

**Review by matyikriszta — APPROVED** [review]

**Pull request merged**